src/hooks/sdk/useSelectAudioInputDevice.tsx

```diff
@@ -1,22 +1,19 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { AudioTransformDevice, Device } from 'amazon-chime-sdk-js';
+import { AudioInputDevice } from 'amazon-chime-sdk-js';
 import { useCallback } from 'react';
 
 import { useMeetingManager } from '../../providers/MeetingProvider';
 
 export const useSelectAudioInputDevice = (): ((
-  device: Device | AudioTransformDevice | null
+  device: AudioInputDevice
 ) => Promise<void>) => {
   const meetingManager = useMeetingManager();
 
-  const selectDevice = useCallback(
-    async (device: Device | AudioTransformDevice | null) => {
-      await meetingManager.selectAudioInputDevice(device);
-    },
-    []
-  );
+  const selectDevice = useCallback(async (device: AudioInputDevice) => {
+    await meetingManager.selectAudioInputDevice(device);
+  }, []);
```

Comment:
Can you add meetingManager in the dependency array here? Also same for useSelectVideoInputDevice